pkg/cmd/release/create/create_test.go

```diff
@@ -1212,6 +1212,7 @@ func TestReleaseCreate_AutomationMode(t *testing.T) {
 			})
 
 			api.ExpectRequest(t, "GET", "/api").RespondWith(rootResource)
+			api.ExpectRequest(t, "GET", "/api/Spaces-1").RespondWith(rootResource)
```

Comment:
I had to pass a spaceID into the mock, which means that the go-octopusdeploy client kicks in and tries to load a spaced root resource... so the unit tests need to satisfy that. sigh (but it's not a big deal, as you can see here)